README.md

```diff
@@ -33,3 +33,27 @@ Consult the help for the latest usage:
 ```console
 $ pyproject-fmt --help
 ```
+
+## Programmatic API
```

Comment:
Oh, please not this. I want sphinx + readthedocs, not an easy-to-go out-of-date markdown blob.

Reply:
Thank you very much for the review @gaborbernat, I am working on the sphinx + readthedocs separatedly in https://github.com/abravalheri/pyproject-fmt/pull/1.

I am assuming that with this comment you prefer that a quick introduction is not added to the README, so I will go ahead and more this example to the function's docstring.

Reply:
> I am working on the sphinx + readthedocs separatedly in [abravalheri#1](https://github.com/abravalheri/pyproject-fmt/pull/1).

I'll take this, because I have a peculiar way of doing things :blush: so I'd not burden you with a lot of back and forth to get there.

Reply:
> I am assuming that with this comment you prefer that a quick introduction is not added to the README, so I will go ahead and more this example to the function's docstring.

I want sphinx documentation generated. This means the sphinx PR needs to go first in before this PR lands.

Reply:
Thank you @gaborbernat, I will wait until your PR lands and then touch this one again.